jenkins/mam4xx_compare_performance.sh

```diff
@@ -112,8 +112,15 @@ main() {
 
     #copy plot to /compyfs/www
     if [ "$mach" = "compy" ]; then
-      cp ${data_dest}/performance_comp_${DATE}_${resolution}.png ${share_dest}/performance_comp_${resolution}.png
-      echo "visit https://compy-dtn.pnl.gov/litz372/performance_data/performance_comp_${resolution}.png for the results!"
+      cp ${data_dest}/performance_comp_${DATE}_${resolution}_${timestep}.png ${share_dest}/performance_comp_${resolution}_${timestep}.png
+      echo "visit https://compy-dtn.pnl.gov/litz372/performance_data/performance_comp_${resolution}_${timestep}.png for the results!"
+    fi
+
+    #copy plot to /cfs/e3sm/www
+    if [ "$mach" = "pm-gpu" ]; then
+      cp ${data_dest}/performance_comp_${DATE}_${resolution}.png ${share_dest}/performance_comp_${resolution}_${timestep}.png
+      chmod 775 ${share_dest}/performance_comp_${resolution}.png
+      echo "visit https://portal.nersc.gov/cfs/e3sm/litz372/performance_data/performance_comp_${resolution}_${timestep}.png for the results!"
```

Comment:
Is it possible to get rid of the machine-specific conditionals in this file? I would like to keep all machine-related stuff in machine files (e.g., `jenkins/perlmutter_mam4xx_compare_performance.sh`). We can store everything that we need in variables and then use those variables in the file. This will also keep everything standardized.

For `chmod`, we can use `chmod` for all machines, as it has to be world-readable to work anyway.